community/membership.md

```diff
@@ -252,23 +252,35 @@ with the current state.
 ### How inactivity is measured
 
 Inactive members are defined as members of one of the Argoproj Organizations
-with **no** contributions across any organization within 12 months. This is
-measured by the CNCF [DevStats project].
+with no significant contributions within the last 12 months. Contributions are
+measured using the CNCF [DevStats project], GitHub activity history, and other media
+by which a history of contributions can be reliably determined.
 
-**Note:** Devstats does not take into account non-code contributions. If a
-non-code contributing member is accidentally removed this way, they may open an
+If an actively contributing member is accidentally removed this way, they may open an
 issue to quickly be re-instated. 
 
-After an extended period away from the project with no activity
-those members would need to re-familiarize themselves with the current state
+After an extended period away from the project with no activity,
+removed members would need to re-familiarize themselves with the current state
 before being able to contribute effectively. 
 
 
 ## Change in membership roles
 
-As described above, a change in membership role may be approved by the appropriate subproject lead if there are no objections from the other leads. By convention, role changes are discussed in a meeting with other approvers before being formally approved.
+A change in membership role may be approved by consensus of the Argo Project leads and approvers. By convention, role changes are discussed in a meeting with Argo Project approvers before being finalized.
 
 
+## Appointment of Subproject Leads
+
+Leads serve a unique and critical role in the direction and organization of each subproject. Each subproject will generally have two co-leads, who share responsibility. There is no fixed term for leads and they often serve until they retire or, in rare cases, are replaced by [voting](GOVERNANCE.md). Leads are encouraged to give other qualified members of the community an opportunity to serve. Former leads are welcome to serve as leads again in the future.
```

Comment:
> Each subproject will generally have two co-leads

Could you clarify whether there is a hard limit on the number of co-leads?

> Former leads are welcome to serve as leads again in the future

What would the process be?

Reply:
The same as for new leads

Reply:
Thanks. What about the question on the number of co-leads?